### PR TITLE
feat: add initial samples and tests

### PR DESCRIPTION
**Pull request by sofisl**

### CODE REVIEW

**Comment by sofisl** [comment]
@bcoe, I started the library off with Node 12 in order to not migrate it in the future. Should I also delete Node 10 configs g3?

File: samples/test/quickstart.js [hunk 1]
```diff
@@ -19,32 +19,30 @@
...
-    // eslint-disable-next-line no-unused-vars
-    projectId = await client.getProjectId();
-  });
-
   it('should run quickstart', async () => {
-    //TODO: remove this line
-    // eslint-disable-next-line no-unused-vars
-    const stdout = execSync('node ./quickstart.js', {cwd});
-    //assert(stdout, stdout !== null);
+    // We should get an error, since we don't have a DNS service
```
Comment (bcoe) [inline samples/test/quickstart.js:33]: Is it possible for us to create a service in the UX of long-door-651, and then test against that?

**Review by JustinBeckwith — APPROVED** [review]
Looks like it just needs the ol `npm run fix`

**Review by bcoe — APPROVED** [review]
@sofisl LGTM once `fix` is run. It might be worth considering moving to OwlBot while we're at it, not  blocker for me though (can be done in follow up work).

**Pull request merged**